cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaVersionTest.java

```diff
@@ -31,6 +31,20 @@ private Reader getKafkaVersionsReader(String kafkaVersions) {
         return new InputStreamReader(KafkaVersion.class.getResourceAsStream("/" + kafkaVersions), StandardCharsets.UTF_8);
     }
 
+    @ParallelTest
+    public void parsingInvalidVersionTest() {
```

Comment:
I'm not sure I get the test => you added new method `compareDottedIVVersion`. But you do not seem to use it in the test. Plus the `compareDottedVersion` which I would expect to fail seems to be passing here for `2.8-IV1`.